content/en/cloudcraft/account-management/cancel-subscription.md

```diff
@@ -3,17 +3,32 @@ title: Cancel your Cloudcraft Subscription
 kind: documentation
 ---
 
-We're sorry to see you go! Only the Account Owner can cancel a Cloudcraft subscription, which takes effect at the end of the current billing cycle. When you cancel your subscription, your account is downgraded to the free plan. 
+Canceling your Cloudcraft subscription is super easy and can be done in a few clicks from inside the application.
```

Comment:
```suggestion
Canceling your Cloudcraft subscription can be done from inside the Cloudcraft application.
```

Reply:
Can you add the exact instructions on how to cancel the subscription? :)

Reply:
They should be a little farther down, I believe.